### ISSUE TITLE
Change register fail message when a user already registered the machine

### ISSUE THREAD

**Issue by niryariv** [hossted/cli#27]
In this situation:
```
ubuntu@gitbucket:~$ hossted register
Email: nir@linnovate.net
Registering user. Please wait a second.
Empty Session Token. Some kind of error occoured - {"msg":"This instance has already been registered under a different owner.","jwt":null,"url":"https://app.hossted.com","success":false}.
```
Display:
`Instance is already registered to another owner. Please contact support@hossted.com if you'd like to add a user`

**Comment by billylkc** [comment]
Should be implemented in v0.1.8

**Comment by billylkc** [comment]
Implemented in 0.1.8, please check

**Comment by niryariv** [comment]
Confirmed to work on v0.1.8.

Please modify text to "Instance is already registered. Please contact support@hossted.com if you'd like to add a user" - so the message is correct also in cases when the user runs hossted register again under the same user

**Comment by niryariv** [comment]
fixed daab39ed97d8d102f2570af44e2a29146137ef91
